pkgs/development/libraries/freeimage/default.nix

```diff
@@ -29,6 +29,10 @@ stdenv.mkDerivation {
 
   preInstall = ''
     mkdir -p $INCDIR $INSTALLDIR
+  ''
+  # Workaround for Makefiles.osx not using ?=
+  + ''
+    makeFlagsArray=( "INCDIR=$INCDIR" "INSTALLDIR=$INSTALLDIR" )
```

Comment:
Shouldn't this only be added if we're building for Darwin?

Reply:
Or you could remove the `+` since it's unconditional.

Reply:
The plus was there to avoid dependency on the contents of the comment.